packages/SwingSet/src/kernel/loadVat.js

```diff
@@ -69,7 +76,7 @@ export function makeVatLoader(stuff) {
   const allowedDynamicOptions = [
     'description',
     'metered',
-    'managerType',
+    'managerType', // TODO: not sure we want vats to be able to control this
```

Comment:
are **TODO**s suppose to survive code review?

Reply:
I'd say yes, and in this case it's capturing a conversation we had externally (rather than being a leftover item of this particular patch).